outport/firehose/firehoseIndexer_test.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestFirehoseIndexer_SaveBlock(t *testing.T) {
+func TestFirehoseIndexer_SaveBlockHeader(t *testing.T) {
```

Comment:
please add an unit test for the constructor with nil writer.

Reply:
Sure, added `TestNewFirehoseIndexer`

Reply:
ok

Reply:
ok